pom.xml

```diff
@@ -577,6 +588,9 @@
                                 <excludes>
                                     <exclude>**/main/**/samples/**</exclude>
                                 </excludes>
+                                <address>localhost</address>
+                                <port>${jacocoPort}</port>
+                                <propertyName>jacocoAgent</propertyName>
```

Comment:
Why is this needed? Why was it not needed before?

Reply:
Currently, ITs do not participate in jacoco coverage. This is the infra to enable this (usage coming in the Jakarta EE PR)

Reply:
removed